gapis/server/server.go

```diff
@@ -152,23 +168,47 @@ func (s *server) GetDevicesForReplay(ctx context.Context, p *path.Capture) ([]*p
 
 	all := bind.GetRegistry(ctx).Devices()
 	filtered := make([]bind.Device, 0, len(all))
-nextDevice:
+nextAndroidDevice:
```

Comment:
Humm okay. The order of devices [is not documented](https://github.com/google/gapid/blob/master/gapis/service/service.go#L70) to have any particular order. We should probably change that. If we do, I'd prefer it if the `GetDevices` RPC kept the same order. 
Finally, can we try to kill the copy-paste code? Adding a for loop or extracting to a function would suffice.